gopom.go

```diff
@@ -24,6 +24,37 @@ func Parse(path string) (*Project, error) {
 	return &project, nil
 }
 
+func ParseFromReader(reader io.Reader) (*Project, error) {
+	b, _ := ioutil.ReadAll(reader)
+	var project Project
+
+	err := xml.Unmarshal(b, &project)
+	if err != nil {
+		return nil, err
+	}
+	return &project, nil
+}
+
+func ParseFromByte(b []byte) (*Project, error) {
```

Comment:
Not necessary, we can construct a io.Reader from bytes through 
`bytes.NewReader(b)` which means we can instead use the `ParseFromReader` function.